src/answers-headless.ts

```diff
@@ -255,6 +415,17 @@ export default class AnswersHeadless {
     });
   }
 
+  /**
+   * Uses {@link AnswersCore} to perform a filtersearch request against specified
+   * fields within a single vertical with the specified parameters and the
```

Comment:
I don't think we need the "with the specified parameters" part, however it might be good to mention that the verticalKey comes from the state.